napari/components/camera.py

```diff
@@ -26,16 +27,22 @@ class Camera(EventedModel):
         sets of Euler angles may lead to the same view.
     perspective : float
         Perspective (aka "field of view" in vispy) of the camera (if 3D).
-    interactive : bool
-        If the camera interactivity is enabled or not.
+    interactive : bool [DEPRECATED]
+        If the camera mouse pan/zoom is enabled or not.
+        Use the mouse_pan and mouse_zoom attributes instead.
```

Comment:
Did this way of render is expected and clear for a potential user?
![obraz](https://user-images.githubusercontent.com/3826210/231301775-28fced94-de84-4729-935a-b955ca27c410.png)

Reply:
Can you please show how it should be specified properly here?

Reply:
It looks like it should be like this `interactive [DEPRECATED] : bool`. Please check if this variant renders correctly,

Reply:
I get this screenshot from downloading build artifacts from this PR, not from build it. But I could do it for you.